EIPS/token_validation.md

```diff
@@ -0,0 +1,179 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Token Validation
+    Author: Brooklyn Zelenka<brooklyn@finhaven.com>, Tom Carchrae<tom@finhaven.com>, Gleb Naumenko<gleb@finhaven.com>
+    Type: Standard Track
+    Category: ERC
+    Status: Draft
+    Created: 2018-02-14
+
+## Simple Summary
+A protocol for services providing token ownership and transfer validation.
+
+## Abstract
+This standard provides a registry contract method for authorizing token transfers.
+By nature, this covers both initially issuing tokens to users (ie: transfer from contract to owner),
+transferring tokens between users, and token spends.
+
+## Motivation
+The tokenization of assets has wide application,
+not least of which is financial instruments such as securities.
+Most jurisdictions have placed legal constraints on what may be traded,
+and who can hold such tokens which are regarded as securities. Broadly this includes KYC and AML validation,
+but may also include time-based spend limits, total volume of transactions, and so on.
+
+Regulators and sanctioned third-party compliance agencies need some way to link
+off-chain compliance information such as identity and residency to an on-chain service.
+The application of this design is broader than legal regulation, encompassing all manner
+of business logic permissions for the creation, management, and trading of tokens.
+
+## Specification
+
+### TokenValidator
+
+```solidity
+interface TokenValidator {
+  function check(
+    address _token,
+    address _address
+  ) public returns(uint8 result)
+
+  function check(
+    address _token,
+    address _from,
+    address _to,
+    uint256 _amount
+  ) public returns (uint8 result)
+}
+```
+
+#### Methods
+
+##### check/2
+
+`function check(address _token, address _user) public returns (uint8 _resultCode)`
+
+> parameters
+> * `_token`: the token under review
+> * `_user`: the user to check
```

Comment:
nit; `_user` isn't the same as `_address` above.  

btw, i kind of agree with the comment you made (notified on slack, but not on github anymore) around `_address` being nebulous.  perhaps `_to` would be a better descriptor?  not certain on that though, but it makes it somewhat consistent with `check/4` and i think it might make sense.  but not sure.

Reply:
I'm not totally certain about `_to`, since the arities are more than optional params. It could also be `_from` or something else. "Bob has surpassed his trade volume limit" or "Alice is a known criminal". 

I think that this is still in line with Solidity, since even with the same arity we could have `foo(uint8 _x)`, `foo(uint8[] _xs)` and `foo(mapping(address => bool) _convenience)`. Of course as a best practice all `foo`s should have some semantic similarity. An implementer would be able to rename a field to better suit their domain, too , since Solidity it only cares about the types. `check(address _bank, address _parentOrGuardian)`.